docs/src/getting_started.dox

```diff
@@ -573,11 +573,8 @@ Each of the options are explained below:
 
 - `USE_DTOA_LIBRARY`: See @ref gsg_dtoa "using a double formatting library".
 
-- `USE_FMEMOPEN`: When inserting an image from a buffer using
-  `worksheet_insert_image_buffer()` libxlsxwriter writes the buffer to a
-  temporary file before processing it. In order to avoid this small overhead
-  you can use this option to invoke `fmemopen()` instead. This option isn't on
-  by default since it isn't supported on Windows.
+- `USE_FMEMOPEN`: Use fmemopen()/open_memstream() instead of temporary files.
```

Comment:
I probably would have used another name here. `USE_FMEMOPEN` was okay for an obscure feature to avoid using files for images but this feature is more general. Maybe use `USE_MEMFILE` or `USE_FILE_BUFFER` instead.  Then have that define `USE_FMEMOPEN` in the Makefile and we can drop the mention of `USE_FMEMOPEN` from the docs (but not from the code).

Reply:
P.S. we will need to do that in the CMake file as well.

Reply:
What do you think about `USE_MEMORY_TMPFILE` to match the other tmpfile option? I think that makes it immediately clear what it does (as opposed to a generic use of memory files).